src/providers/postgres/qgspostgresconn.h

```diff
@@ -546,7 +546,7 @@ class QgsPostgresConn : public QObject
     bool mSwapEndian;
     void deduceEndian();
 
-    int mNextCursorId;
+    static int mNextCursorId;
```

Comment:
I think this will need to be QAtomicInt, or something else with thread safety...

Reply:
Thanks for the tip.